src/pages/products/helper/CartHelper.js

```diff
@@ -1,32 +1,81 @@
-import {
-  ADD_TO_CART,
-  DECREASE_QUANTITY,
-  INCREASE_QUANTITY,
-  REMOVE_FROM_CART,
-} from "../../../utils/Constant";
+import axios from "axios";
+import { UPDATE_CART_DATA } from "../../../utils/Constant";
+import { getUserToken } from "../../../utils/TokenHelper";
+import { addToWishlist } from "./WishlistHelper";
 
-export const addToCart = (item, dispatch) => {
-  dispatch({
-    type: ADD_TO_CART,
-    payload: item,
-  });
+export const addToCart = async (item, dispatch) => {
+  try {
+    const token = getUserToken();
+    if (token) {
+      const { status, data } = await axios.post(
+        "/api/user/cart",
+        {
+          product: item,
+        },
+        {
+          headers: { authorization: token },
```

Comment:
you can put headers object globally so that in each function you dont have to setup headers

Reply:
Ok, will try that